org.adempiere.base/src/org/compiere/model/MProduct.java

```diff
@@ -48,6 +48,7 @@
  * @author Mark Ostermann (mark_o), metas consult GmbH
  * 			<li>BF [ 2814628 ] Wrong evaluation of Product inactive in beforeSave()
  */
+@org.adempiere.base.Model(table="M_Product")
 public class MProduct extends X_M_Product implements ImmutablePOSupport
 {
 	/**

```

Comment:
don't need an example here unless it is an modification for all M* classes

Reply:
Please see below.